bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+import Button from '/imports/ui/components/button/component';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { styles } from './styles';
+
+const intlMessages = defineMessages({
+  downloadPresentationButton: {
+    id: 'app.downloadPresentationButton.label',
+    description: 'Download presentation label',
+  },
+});
+
+const propTypes = {
+  intl: intlShape.isRequired,
+  handleDownloadPresentation: PropTypes.func.isRequired,
+  dark: PropTypes.bool,
+  elementName: PropTypes.string,
+};
+
+const defaultProps = {
+  dark: false,
+  elementName: '',
+};
+
+const DownloadPresentationButton = ({
+  intl, handleDownloadPresentation, dark, elementName,
+}) => {
+  const formattedLabel = intl.formatMessage(
+    intlMessages.downloadPresentationButton,
+    ({ 0: elementName ? elementName.toLowerCase() : '' }),
+  );
+
+  return (
+    <div className={cx(styles.wrapper, dark ? styles.dark : styles.light)}>
+      <Button
```

Comment:
Why don't use a link with download attribute?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Reply:
We could. I saw we do that for downloading chat. However here I don't have much to pass in term of data. I don't know the mime type, I want to preserve the name, etc... Is there an issue with the Button? I do uriEncode the params